internal/cmd/iam/command_groupmapping_create.go

```diff
@@ -0,0 +1,56 @@
+package iam
+
+import (
+	"github.com/spf13/cobra"
+
+	ssov2 "github.com/confluentinc/ccloud-sdk-go-v2/sso/v2"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+)
+
+func (c *groupMappingCommand) newCreateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create <name>",
+		Short: "Create a group mapping.",
+		Args:  cobra.ExactArgs(1),
+		RunE:  c.create,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Create a group mapping named "DemoGroupMapping".`,
+				Code: `confluent iam group-mapping create DemoGroupMapping --description "description of group mapping" --filter "\"demo\" in claims.group"`,
+			},
+		),
+	}
+
+	cmd.Flags().String("description", "", "Description of the group mapping.")
+	cmd.Flags().String("filter", "", "CEL-compliant filter for this group mapping")
+	pcmd.AddOutputFlag(cmd)
+
+	return cmd
+}
+
+func (c *groupMappingCommand) create(cmd *cobra.Command, args []string) error {
+	name := args[0]
```

Comment:
IMO, no need to make a variable here, especially since this only gets used in one spot.